fontforge/python.c

```diff
@@ -19347,34 +19342,37 @@ extern void PyFF_FreePythonPersistent(void *python_persistent) {
     Py_XDECREF((PyObject *)python_persistent);
 }
 
+static gint GPtrArrayStrcmp(gconstpointer a, gconstpointer b) {
+    return strcmp(*(const char**)a, *(const char**)b);
+}
+
 static void LoadFilesInPythonInitDir(char *dir) {
     DIR *diro;
     struct dirent *ent;
-    struct string_list *filelist=NULL;
-    struct string_list *item;
+    GPtrArray *filelist;
 
     diro = opendir(dir);
     if ( diro==NULL )		/* It's ok not to have any python init scripts */
 return;
 
+    filelist = g_ptr_array_new_with_free_func(free);
```

Comment:
I would recommend avoiding squashing these substantial python changes at merge. Keeping the commits separate would presumably be fine.

Reply:
Yeah, usually I squash anything I want to squash before merging, and merge PRs as-is. I'm leaving the removal commits separate to the larger refactor commit.